### PR TITLE
Minor sandcastle cleanup

### PR DESCRIPTION
**Pull request by hpinkos**
While going through examples, I found a few things that I thought could be improved

I reset the camera position between demos in `Billboards` and `CZML`, made `3D Models` 3D only, and sped up clock in the `CZML Polygon - Intervals, Availability.html` so you didn't have to wait forever to see a change.

### CODE REVIEW

**Comment by bagnell** [comment]
You should merge the `cameraFlyHome` branch from #3506 here and use that instead of the home button. Then I'll just merge them in order.

File: Apps/Sandcastle/gallery/3D Models.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "use strict";
 //Sandcastle_Begin
 var viewer = new Cesium.Viewer('cesiumContainer', {
+    scene3DOnly: true,
```
Comment (emackey) [inline Apps/Sandcastle/gallery/3D Models.html]: Maybe turn off the scene mode picker here?

Comment (hpinkos) [inline Apps/Sandcastle/gallery/3D Models.html]: The scene mode picker is turned off automatically if `scene3DOnly` is `true`

**Comment by hpinkos** [comment]
@bagnell ready

**Pull request merged**